### PR TITLE
(fix) Account versioning pytest fix

### PR DESCRIPTION
**Pull request by EgorKulikov**
Fixes newly failed tests in nayduck

### CODE REVIEW

**Review by bowenwang1996 — APPROVED** [review]
Thanks. Could you also provide a link to some nayduck run here?

**Comment by mikhailOK** [comment]
http://nayduck.eastus.cloudapp.azure.com:3000/#/test/119704 one more test

**Comment by bowenwang1996** [comment]
@EgorKulikov Please fix the test that @mikhailOK mentioned before merging your PR

**Comment by mikhailOK** [comment]
`pytest sanity/sync_chunks_from_archival.py --features nightly_protocol --features nightly_protocol_features` http://nayduck.eastus.cloudapp.azure.com:3000/#/test/120005 - gets past genesis but still fails
`pytest sanity/transactions.py --features nightly_protocol --features nightly_protocol_features` http://nayduck.eastus.cloudapp.azure.com:3000/#/test/119969 - could be unrelated but looks suspicious

**Comment by EgorKulikov** [comment]
> `pytest sanity/transactions.py --features nightly_protocol --features nightly_protocol_features` http://nayduck.eastus.cloudapp.azure.com:3000/#/test/119969 - could be unrelated but looks suspicious

this one failed simillary before, see http://nayduck.eastus.cloudapp.azure.com:3000/#/test/115535
so it seems like this is just flaky

**Comment by mikhailOK** [comment]
with #4135 transactions.py should be passing: http://nayduck.eastus.cloudapp.azure.com:3000/#/run/1378

**Comment by EgorKulikov** [comment]
> with #4135 transactions.py should be passing: http://nayduck.eastus.cloudapp.azure.com:3000/#/run/1378

Sorry, I think I've just ran that on unmerged master, that's the reason. I've restarted after merging

**Comment by EgorKulikov** [comment]
> `pytest sanity/sync_chunks_from_archival.py --features nightly_protocol --features nightly_protocol_features` http://nayduck.eastus.cloudapp.azure.com:3000/#/test/120005 - gets past genesis but still fails

Seems like flaky test tbh - http://nayduck.eastus.cloudapp.azure.com:3000/#/test_history/120391

**Comment by mikhailOK** [comment]
http://nayduck.eastus.cloudapp.azure.com:3000/#/test_history/118850 - looks less flaky before #4089, but maybe nayduck is slow today.

**Comment by EgorKulikov** [comment]
> with #4135 transactions.py should be passing: http://nayduck.eastus.cloudapp.azure.com:3000/#/run/1378

passed after merging

**Comment by bowenwang1996** [comment]
@EgorKulikov is this ready to be merged?

**Comment by EgorKulikov** [comment]
@bowenwang1996 I've made some additional changes since review, which probably need to be reviewed

File: test-utils/testlib/src/node/process_node.rs [hunk 1]
```diff
@@ -138,17 +138,19 @@ impl ProcessNode {
...
+                args.push("nightly_protocol");
+            }
+            #[cfg(feature = "nightly_protocol_features")]
+            {
+                args.push("--features");
+                args.push("nightly_protocol_features");
+            }
+            args.append(&mut vec!["--bin", "neard", "--", "--home", &self.work_dir, "run"]);
+            command.args(&args[..]);
             command
```
Comment (matklad) [inline test-utils/testlib/src/node/process_node.rs:154]:
```suggestion
            command.args(&["run", "-p", "neard"]);
            #[cfg(feature = "nightly_protocol")]
            command.args(&["--features", "nightly_protocol"]);
            #[cfg(feature = "nightly_protocol_features")]
            command.args(&["--features", "nightly_protocol_features"]);
            command.args(&["--bin", "neard", "--", "--home", &self.work_dir, "run"]);
            command
```

This is a more direct way to express this, you can call `.args` or `.args` on a `Command` several times.

**Pull request merged**